### PR TITLE
Build and publish with GitHub Actions

### PR DESCRIPTION
**Pull request by Matyrobbrt**
The templates have been generated using the GU `setupGitHubActionsWorkflows` task.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -30,48 +30,36 @@ java {
 }
 
 changelog {
-    fromTag "0.1"
+    from '1.1'
+}
+
+test {
```
Comment (Technici4n) [inline build.gradle:36]: ?

Comment (Matyrobbrt) [reply build.gradle:36]: There are no tests, so there's no reason for Gradle to even attempt to run the test task during `build`

Comment (Technici4n) [reply build.gradle:36]: You can leave it though, right? Or will it cause issues?

Comment (Matyrobbrt) [reply build.gradle:36]: Enabling it won't cause issues, but neither will disabling it.

Comment (Technici4n) [reply build.gradle:36]: Just leave it enabled then, it's less code... :P

Comment (Matyrobbrt) [reply build.gradle]: I... don't feel like arguing, so whatever.

**Review by Su5eD — APPROVED** [review]

**Pull request merged**